api/src/main/java/io/jsonwebtoken/JwtParserBuilder.java

```diff
@@ -132,6 +140,16 @@
      */
     JwtParserBuilder require(String claimName, Object value);
 
+    /**
+     * Require the payload of the JWT not to be empty. The default value is false. If a message contains an empty payload
+     * string but it is required per this flag, parsing the JWS will lead to a {@link MalformedJwtException}.
+     *
+     * @param payloadRequired
+     * @return the parser builder for method chaining.
+     * @see MalformedJwtException
+     */
+    JwtParserBuilder requirePayload(boolean payloadRequired);
```

Comment:
We could probably remove the arg and make this just `requirePayload()`.  Assuming the default is always "false".  
@lhazlewood ?